code/modules/mob/living/carbon/superior_animal/robots/subtype/greyson/types/turrets.dm

```diff
@@ -78,6 +78,69 @@
 
 
 /mob/living/carbon/superior_animal/robot/greyson/stalker/dual/New()
-	..()
-	if(prob(1))
+	. = ..()
+	if(prob(1) && (!drop2))
 		drop2 = /obj/item/gun/projectile/automatic/scaffold
+
+// mini-boss enemy that demands attention or else they will nuke someone. VERY POWERFUL
+/mob/living/carbon/superior_animal/robot/greyson/stalker/dual/plasma_cannon
+	name = "\"Iron Lock Security\" Assault Stalker Mk2"
+	desc = "A ruthless patrol borg that defends Greyson facilities. This one has a pair of massively oversized plasma cannons, and has been fitted with thick layers of ablative plating, although at the cost of it's mobility, melee, and ballistic defenses. This will destroy you."
+
+	range_charge_telegraph = "'s plasma cannons grow brighter, and it hums louder, preparing to fire at"
+	range_telegraph = "'s plasma cannons let out an eerie and TERRIFYING whine as it prepares to unleash it's devastating payload upon"
+	target_telegraph = "begins to hum, it's plasma cannons glowing with a dim, growing light, as it turns to"
+
+	telegraph_beam_color = COLOR_RED
+	color = COLOR_RED
+
+	rounds_left = 4
+	mag_type = /obj/item/cell/large/hyper/depleted
+	mags_left = 1
+
+	advance = FALSE
+
+	armor = list(melee = 10, bullet = 15, energy = 60, bomb = 100, bio = 100, rad = 100) //takes stalker armor to the extreme, also, if people want to melee the stalker that explodes apon death, power to them
+
+	maxHealth = 470 //high hp is required due to it bombing itself a lot
+	health = 470
+
+	deathmessage = "violently explodes, it's internal plasma cells combusting along with it's remaining cells!"
+	reload_message = "lets out a hiss as it ejects a cell from it's carapace!"
+
+	projectiletype = /obj/item/projectile/hydrogen/cannon/max //devastating
```

Comment:
GP lore donst allow them to have SI hydrogen tech

Reply:
Maybe its better to have it say its an RPG and have it drop an rpg?
Just spitballing or something

Reply:
i mean, its only using the /projectile/

if a maintainer doesnt like it i can always just change the projectile name, but, theres literally no outward way for the players to know im using a hydrogen bolt, since its called a plasma bolt ingame, and plasma != hydrogen

Reply:
Plasma's fine.